Adding Web mockup (unfinished)

AninOnin opened the pull request:
Adding unfinished web presence for app, just trying to get it to a working version, then will polish up.

Commit:
Adding Web mockup (unfinished)

Adding unfinished web presence for app, just trying to get it to a
working version, then will polish up.

rhefner1 reviewed (changes requested):
Looking good so far, but we need changes in (at least) the following files:
 - `app.yaml`
 - `http_handler.py`

Commit:
Modified handlers, moved folders into cookomatic_web

Altered app.yaml, http_handler.py, moved assets and templates folder
into cookomatic_web folder.

Commit:
Altered index.html

should work now

Commit:
i promise it'll work this time
